prov/efa/src/rxr/rxr_pkt_cmd.c

```diff
@@ -312,7 +312,16 @@ ssize_t rxr_pkt_post_ctrl_once(struct rxr_ep *rxr_ep, int entry_type, void *x_en
 	if (!pkt_entry)
 		return -FI_EAGAIN;
 
-	pkt_entry->send = malloc(sizeof(struct rxr_pkt_sendv));
+	/*
+	 * We have to carry the allocated rxr_pkt_sendv strcut until the corresponding
+	 * pkt_entry gets released, because the pkt_entry could be queued and re-send
+	 * later (e.g, in case of RNR cq error happens)
```

Comment:
I do not quite understand the comments

Reply:
I do not think the comment is really necessary.

Reply:
hmm, I'm inclined to keep it, as this is where we were missing and resulted in a bug. But I can remove it if you insist.

Reply:
maybe just explain that `pkt_entry->send` will be released in when releasing packet entry. also there is a typo `strcut`

Reply:
Thanks.